pegasus/sites.v3/hourofcode.com/views/join_us_description.haml

```diff
@@ -3,19 +3,21 @@
     %h2{:style=>"margin-top: 0px"}
       =hoc_s(:front_join_us_header)
   #calltoaction
-    %ul{:style=>"line-height:28px"}
-      %li
-        %b=hoc_s(:front_join_us_teacher)
-        =hoc_s(:front_join_us_inspire)
-        %a{href:resolve_url('/resources/how-to')}
-          =hoc_s(:front_join_us_inspire_link)
-      %li
-        %b=hoc_s(:front_join_us_parent)
-        %a{href:resolve_url('/resources')}
-          =hoc_s(:front_join_us_ask_link)
-        =hoc_s(:front_join_us_ask)
-      %li
-        %a{href:resolve_url('/resources')}
-          =hoc_s(:front_join_us_help_promote)
+    %div
+
+      %a.category-link{:href=>resolve_url('/resources/how-to')}
+        %div.category-button{:style=>"background-color: #59b9dc"}
+          %b.category-header=hoc_s(:front_join_us_teacher)
+          %span=hoc_s(:front_join_us_teacher_link)
+      %a.category-link{:href=>resolve_url('/resources/promote#sample-emails')}
+        %div.category-button
+          %b.category-header=hoc_s(:front_join_us_parent)
+          %span=hoc_s(:front_join_us_parent_link)
+      %a.category-link{:href=>resolve_url('/resources/promote')}
+        %div.category-button{:style=>"background-color: #d1d567"}
+          %b.category-header=hoc_s(:front_join_us_computer_science)
+          %span=hoc_s(:front_join_us_spread_word)
       -if chapter_partner?
-        %li=hoc_s(:front_join_us_more_info).gsub('code.org', resolve_url('code.org'))
+        %a.category-link{:href=>resolve_url('https://code.org')}
+          %div.category-button{:style=>"background-color: #ffc459"}
```

Comment:
I think the colors are too light, and it's just a wee bit too hard to read. I think it needs to be the regular colors, and switch this last one from orange to teal. 

![image](https://cloud.githubusercontent.com/assets/1920530/9890895/a32f1082-5bb7-11e5-8084-a41079ad2b21.png)

![image](https://cloud.githubusercontent.com/assets/1920530/9890861/81b617a2-5bb7-11e5-88dc-da992de28b26.png)